Jenkinsfile

```diff
@@ -26,7 +26,7 @@
 
 
 properties([
-    disableConcurrentBuilds(),
+    disableConcurrentBuilds(abortPrevious: true),
```

Comment:
Wasn't there an issue with enabling this? @msarahan would know?

Reply:
I think it should be fine. It's a multi-branch pipeline, and this implies that no more than one build can be happening per-branch, but separate branches may still run concurrently. I think that's fine, but I don't know this pipeline well.

Reply:
Since this is a multi-branch pipeline, we won't run into the same issues we saw months ago on a single branch pipeline.